js/bitcoincoid.js

```diff
@@ -256,16 +255,16 @@ module.exports = class bitcoincoid extends Exchange {
     }
 
     async fetchOpenOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
-        if (!symbol)
-            throw new ExchangeError (this.id + ' fetchOpenOrders requires a symbol');
         await this.loadMarkets ();
-        let market = this.market (symbol);
-        let request = {
-            'pair': market['id'],
-        };
+        let market = symbol ? this.market (symbol) : undefined;
+        let request = market ? { 'pair': market['id'] } : {};
```

Comment:
it won't like these two lines:

```
let market = undefined;
if (symbol)
    market = this.market (symbol);
```

```
let request = {};
if (market)
    request['pair'] = market['id'];
```

Reply:
How do these differ from this line?
https://github.com/ccxt/ccxt/blob/f56b5c2f94a52f4bf5da013c5c545273856ab2be/js/liqui.js#L474

Reply:
this.market() has round brackets, this breaks the transpiler, as round brackets add ambiguity (very hard to determine when one part of the expression starts and the other ends with pure regexes like that).